controllers/controllers.js

```diff
@@ -17,3 +21,15 @@ exports.getArticleById = (req, res, next) => {
 			next(err)
 		})
 }
+
+exports.deleteCommentById = (req, res, next) => {
+	const { comment_id } = req.params
+
+	removeCommentById(comment_id)
+		.then((deletedComment) => {
+			res.status(204).send()
```

Comment:
A little shorthand syntax you can use when not sending any content is 
`res.sendStatus(204);`